ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -124,6 +124,13 @@ public int execute() {
     try {
       long loadTaskStartTime = System.currentTimeMillis();
       SecurityUtils.reloginExpiringKeytabUser();
+      //Don't proceed if target db is replication incompatible.
+      if (work.dbNameToLoadIn != null) {
+        Database targetDb = getHive().getDatabase(work.dbNameToLoadIn);
+        if (targetDb != null && MetaStoreUtils.isDbReplIncompatible(targetDb)) {
```

Comment:
When would targetDb be null? When a wrong work.dbNameToLoadIn is specified ?

Reply:
for the first load operation (bootstrap load), targetDb would be null.